.github/workflows/test-nightly.yml

```diff
@@ -36,17 +40,21 @@ jobs:
           fi
 
       - name: Build Docker image locally
-        if: ${{ !inputs.tag }} # only build an image if no tag is specified
+        if: ${{ inputs.tag == '' }} # only build an image if no tag is specified
```

Comment:
why did this change?

Reply:
b/c gh Actions treats inputs as strings, using `!` on a string is unreliable. The explicit check `inputs.tag == ''` is more robust way to detect if no tag is provided.